specification/06-arch_overview.adoc

```diff
@@ -43,7 +45,12 @@ The device attestation process is specific to each TVM workload configuration,
 but it may be handled by the Trusted Device Manager (TDM).
 
 In order to protect the confidentiality and integrity of TVM data between the
-PCIe root port and a bound TDI, CoVE-IO relies on the TSM to configure both the
-PCIe physical device and the root port it's attached to in order to establish
-IDE streams between the two. The PCIe root port IDE configuration from the TSM
-is abstracted through the platform Root-of-Trust (RoT).
+PCIe root port and a bound TDI, PCIe IDE streams are established between them.
+CoVE-IO relies on the TSM to generate and configure the IDE keys for both the
+PCIe physical device and the PCIe root port the device is attached to. For
+setting the device IDE keys up, the TSM must use the IDE Key Management
+(`IDE_KM`) protocol. For the PCIe root port, the TSM must protect the IDE keys
+exchange through e.g. a SPDM session with the platform Root-of-Trust (RoT) DOE
+mailbox, when it supports that mechanism. If the communication between TSM and
```

Comment:
For this you added the variant where the TSM programs the PCie root directly- does that PR supersede this one?

Reply:
sg